pycaching/cache.py

```diff
@@ -153,6 +153,45 @@ def _from_print_page(cls, geocaching, guid, soup):
         cache_info["log_counts"] = Cache._get_log_counts_from_print_page(soup)
         return Cache(geocaching, **cache_info)
 
+    @classmethod
+    def _from_api_record(cls, geocaching, record):
+        """Create a cache instance from a JSON record returned by API."""
+        assert record['cacheStatus'] in {0, 1}
```

Comment:
We should probably provide a comment about the meaning of the values here.

Reply:
ok, I have defined new enum `Status`